Assets/Scripts/Framework/TargetSelector.cs

```diff
@@ -0,0 +1,37 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Framework
```

Comment:
```suggestion
namespace Framework.GeoLocation
```
Dit script kan in die subfolder. Past daar wel mooi in.